DataProcessing/config.json

```diff
@@ -1,5 +0,0 @@
-{
-    "data-folder": "../../../Data/",
```

Comment:
> "data-folder"

Is a config required by the data processor, suggest leaving the config.json with the needed configurations